### PR TITLE
doc: Add hugo-lyra search engine

### PR DESCRIPTION
**Pull request by paolomainardi**
Hi everyone!

As the title suggests, I am adding a new library I am working on to integrate [Lyra search](https://github.com/LyraSearch/lyra) into a Hugo website.

I wrote this post to explain a bit more in depth: https://www.paolomainardi.com/posts/hugo-search-engine-lyra/

### CODE REVIEW

File: content/en/tools/search.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ A static website with a dynamic search function? Yes, Hugo provides an alternati
 * [hugofastsearch](https://gist.github.com/cmod/5410eae147e4318164258742dd053993). A usability and speed update to "GitHub Gist for Fuse.js integration" — global, keyboard-optimized search.
 * [JS & Fuse.js tutorial](https://makewithhugo.com/add-search-to-a-hugo-site/) A simple client-side search solution, using FuseJS (does not require jQuery).
 * [Pagefind](https://github.com/cloudcannon/pagefind). A fully static search library that aims to perform well on large sites, while using as little of your users' bandwidth as possible.
+* [Hugo Lyra](https://github.com/paolomainardi/hugo-lyra). Hugo-Lyra is a javascript module to integrate [Lyra](https://github.com/LyraSearch/lyra) into a Hugo website. It contains the server-side part to generate the index and the client-side library (optional) to bootstrap the search engine easily.
```
Comment (coliff) [inline content/en/tools/search.md:30]: Looks good! Can you please fix capitalization of 'JavaScript'.

Comment (paolomainardi) [reply content/en/tools/search.md]: @coliff done, thanks!

**Comment by paolomainardi** [comment]
Hi everyone, any chance to have this merged anytime soon ?

**Comment by jmooring** [comment]
Sure, I'll merge it for now, but it may be removed in the future.
See <https://github.com/gohugoio/hugoDocs/issues/1573>

**Comment by paolomainardi** [comment]
Thanks @jmooring 👍

**Pull request merged**